test/verify/check-system-info

```diff
@@ -193,6 +193,12 @@ class TestSystemInfo(MachineCase):
         b.wait_in_text('#system_information_hostname_text', "Adventure Box (host1.cockpit.lan)")
         self.assertEqual(m.execute("hostname").strip(), "host1.cockpit.lan")
 
+        transient_hostname = m.image.split('-')[0]
+        m.execute("hostnamectl set-hostname ''")
```

Comment:
This sets the static *and* transient name, which makes the test pointless. Instead I suggest to do `set-hostname --static ''` and `set-hostname --transient mydhcpname`, so that you pin down the names at that point. Then the actual check can wait for 'mydhcpname'.